php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -577,26 +576,15 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                         $Group['Elements'] = [];
                     }
                     break;
-
                 case 'begintable':
-                    if ($addElements) {
-                        $this->form->addElement(
-                            'static',
-                            '',
-                            "</td></tr></table><table><tr><td>&nbsp;",
-                            ""
-                        );
-                    }
-                    break;
                 case 'endtable':
-                    if ($addElements) {
-                        $this->form->addElement(
-                            'static',
-                            '',
-                            "</td></tr></table><table><tr><td>&nbsp;",
-                            ""
-                        );
-                    }
+                    error_log(
+                        "DEPRECATION MESSAGE: The use of `begintable` and ".
+                        "`endtable` elements of LINST instruments has is no longer ".
+                        "necessary. The $this->testName LINST instrument should ".
+                        "be modified to remove these elements as they will be ".
+                        "completely deprecated in a future version of LORIS"
```

Comment:
I don't think this error message is good.

1. There's a typo
2. It's not that they're no longer necessary, it won't just magically work right if you delete the, it's that they're no longer supported..

Reply:
it does work right if you delete them.

basically without them, everything is fine. with them everything is broken!

Reply:
How about something like:

```
`begintable` and `endtable` elements are deprecated.
$this->testName LINST instrument should be
 modified to remove these elements as they will be
completely removed in a future version of LORIS.
```